### PR TITLE
11 implement crawler

### PR DESCRIPTION
**Pull request by DmytroTHR**
still have issues with site duplication when crawling in parallel mode

### CODE REVIEW

File: cmd/app/crawler.go [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+	"errors"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strings"
+	"sync"
+	"time"
+)
```
Comment (PavelAD) [inline cmd/app/crawler.go:12]: use `go import`

Comment (DmytroTHR) [reply cmd/app/crawler.go:12]: You mean imports order, right?

Comment (PavelAD) [reply cmd/app/crawler.go:12]: yes, firstly standards libs and after from another source
like this
Screenshot 2022-06-27 at 16 23 38

Comment (DmytroTHR) [reply cmd/app/crawler.go:12]: OK, done. Thnx.

**Pull request merged**